sdk/rust-examples/fd_create/rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2021-11-01
```

Comment:
Is it necessary here to force a potentially different version of rust?

Reply:
It's the nightly equivalent. https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1561-2021-11-01 :
"Version 1.56.1 (2021-11-01)"

We need nightly for the `wasm_import_module` stuff.

Reply:
Though bizarrely I can't now get an error if I remove that file so I'll try without it ...